### PR TITLE
DOC-6684 Workloads updates

### PR DESCRIPTION
**Pull request by urbiz-nr**
- Minor tweaks in text in Intro doc
- New image

### CODE REVIEW

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx [hunk 1]
```diff
@@ -9,30 +9,31 @@ redirects:
...
+![workload_activity_tab.png](./images/workloads_activity_tab.png "workloads_activity_tab.png")
 
-3. **Edit workload.** Edit the workload to define the entities that are part of the workload and to link dashboards to the workload.
+Here's the most important sections:
+1. Linked dashboards. You can add links to dashboards from your workload, and [create pre-filtered, workload-relevant links to dashboards](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#add-dashboards).
+2. Golden metrics. These are charts with the most relevant metrics for each entity type, such as number of requests, response time and error rate for an application. Explore the charts to detect correlations among different entities (for example, two applications) and different stack layers (for example, applications and hosts). The golden metrics that you see for each entity type on a workload can be customized either at the account or the workload level through the [NerdGraph API](/docs/apis/nerdgraph/examples/golden-metrics-entities-nerdgraph-api-tutorial/#customize-golden-metrics).
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx:29]: comma after "response time"?

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx [hunk 2]
```diff
@@ -15,17 +15,17 @@ redirects:
...
 
 <Callout variant="tip">
-  To use workloads with Alerts and Applied Intelligence, as well as the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
+  To use workloads and the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
 ## What is a workload in New Relic? [#definition]
 
 New Relic monitors a wide range of entities and data, from client-side applications and back-end APIs, to the underlying infrastructure. To make sense of this large data set, we give you the ability to create and monitor workloads.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:26]: maybe change to "backend" without dash like previous

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx [hunk 3]
```diff
@@ -15,17 +15,17 @@ redirects:
...
 
-Workloads give you the ability to group and monitor entities based on a team or a set of responsibilities, from front-end to back-end services, across your entire stack.
+Workloads give you the ability to group and monitor entities based on a team or a set of responsibilities, and provides an aggregated view of the health and activity of the entities in the workload. Thus, you can understand better how your business logic is working, from front-end to back-end services, across your entire stack.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:28]: "and provide" no s

Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:28]: consistency throughout doc "front-end" "back-end" or "frontend" "backend"

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx [hunk 4]
```diff
@@ -35,10 +35,10 @@ Here are some workload examples:
 
 Here's a workload:
 
-![Operational_workload.png](./images/Operational_workload.png "Operational_workload.png")
+![1_workload_health_tab.png](./images/1_workload_health_tab.png "workload_health_tab.png")
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com) > Apps > Workloads > (selected workload):** The workloads UI provides a curated view of how the entities in your workload are performing. The charts you see will depend on the types of entities you've included to the workload.
+  **[one.newrelic.com](https://one.newrelic.com) > Explorer > Workloads > (selected workload):** The workloads UI provides a curated view of how the entities in your workload are performing. The charts you see will depend on the types of entities you've included to the workload.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:41]: "included in" or "added to" the workload?

**Review by barbnewrelic — APPROVED** [review]
Looks great, Anna! I made some very minor copyedit suggestions. Take 'em or leave 'em! :-)

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx:29]: Thanks!

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx (hunk 3 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:28]: Right!

File: src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx (hunk 4 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx:41]: Done!

**Pull request merged**